src/plugins/gui/providers/bityProvider.ts

```diff
@@ -430,10 +430,59 @@ export const bityProvider: FiatProviderFactory = {
           })
         }
 
-        // Check for a max amount limit
+        // Because Bity only support <1k transactions w/o KYC and we have no
+        // way to KYC a user, add a 1k limit
+        if (amountType === 'fiat') {
+          if (gt(exchangeAmount, '1000')) {
```

Comment:
Does this need to be `gte`? Fix the comment if we only need `gt`

Reply:
comment fixed